ledger/controllers/surveyor.js

```diff
@@ -36,10 +37,45 @@ const server = async (request, reply, runtime) => {
     surveyor.payload = entry.payload
     surveyor.parentId = entry.parentId
   }
+  return surveyor && addChoices(runtime, surveyor)
+}
 
+async function addChoices (runtime, surveyor) {
+  const payload = surveyor.payload || {}
+  surveyor.payload = payload
```

Comment:
Do we need this line?  It looks like all the surveyors have a payload
```
db.surveyors.find({payload: {$exists: false}}).count()
> 0
```

Reply:
probably not it was just me being defensive i suppose

Reply:
not a big deal, was just curious